helix-term/src/ui/statusline.rs

```diff
@@ -254,6 +257,14 @@ where
     );
 }
 
+fn render_primary_selection_length<F>(context: &mut RenderContext, write: F)
+where
+    F: Fn(&mut RenderContext, String, Option<Style>) + Copy,
+{
+    let tot_sel = context.doc.selection(context.view.id).primary().len();
```

Comment:
I know this PR is only concerned with the primary selection, but IMO it might be worthwhile to sum up every range's lengths to also support multi-selection counts. This could either be a separate status line element all together or it could switch to showing the primary and total character count when a multiline selection is active.

<img width="789" alt="image" src="https://user-images.githubusercontent.com/31227862/201047522-6623b2d4-6284-4a07-b954-b6f39b862a0d.png">

That's of course just a side note and this change would 100% solve the original issue

Reply:
i'll leave this effort for another time

Reply:
True that sounds very reasonable 👌